ramp4-pcar4/ramp4-pcar4
# MIL Child Filter Race Fix

### Related Item(s)

#2395

### Changes

- Tweaks code to stop "does layer exist" checks from stopping filters getting applied to freshly born MIL Child layers. It's a special case in that the RAMP layer is pre-initialized but the layer is guaranteed to exist due to it being a child (parent is already in loading phase).
- Changed the fake version of GeoMet layer back to have all of Ontario, for automation testing team.

### QA Testing

Please test against `main` branch https://ramp4-pcar4.github.io/ramp4-pcar4/main/demos/qa-samples.html?sample=2

### Testing

Steps:
1. Visit QA Sample 2. (Can get to these via yellow pages, or steal page name from above url)
2. See that the layers from the "Map Image Layer" section of the legend only have a few points and do not cover all of Canada
3. Mash `F12`. See that the console is not full of errors about setting filters on un-initialized layers. 🦉


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2398)
